### DISCUSSION TITLE
164

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#114] [General]
# 潮流周刊第164期 - 又是一年

封面图来源于天气好的时候在玉鸟集拍摄，第一次过年呆在杭州，感觉这个春节很长的样子，又是新的一年，祝你的心情和这张图片一样舒服。

https://weekly.tw93.fun/posts/164-%E5%8F%88%E6%98%AF%E4%B8%80%E5%B9%B4

**Comment by CY-Christin** [comment]
那个主题有点小问题，标签页之间分割线，标签页开的多的话想选中某个标签不太方便，希望可以优化一下。

**Reply by CY-Christin** [reply]
update:
「标签页之间分割线」 => 「标签页之间没有分割线」

**Reply by huccihuang** [reply]
同意，且使用主题后难以分辨我在哪个标签页

**Comment by la-aa** [comment]
好奇 up 的文章首图是什么设备拍摄的，这么清晰，并且构图色彩都很好，我一度以为是相机，但相机带出门的频率远不如手机。如果是手机的话，拍摄水平很高啊

**Reply by tw93** [reply]
几乎都是 iPhone 15 pro
